stylesheets/main.css

```diff
@@ -25,6 +25,12 @@
 	height: 80px;
 }
 
+.carousel-inner > .item > img,
```

Comment:
Instead of writing it like `carousel-inner > .item > img` we are traversing the nodes in DOM tree without any reason.
Best approach :  add a class to the image and access it in css :)